bundle.yml

```diff
@@ -123,6 +124,14 @@ options :
         cmake: >
             ENABLE_SYCL=ON
 
+    - sycl-sub-group-size :
```

Comment:
The SYCL variants don't like non-matching NGPTOT, NPROMA and SUB_GROUP_SIZE values, right? I'm seeing errors like:
```
$ bin/dwarf-cloudsc-scc-sycl 1 100 32
Running on NVIDIA A100-SXM4-40GB
terminate called after throwing an instance of 'sycl::_V1::nd_range_error'
  what():  Non-uniform work-groups are not supported by the target device -54 (PI_ERROR_INVALID_WORK_GROUP_SIZE)
Aborted
```

This affects, in particular, also CTest where we (intentionally) use a non-power-of-2 NGPTOT. So, I suggest to add a check after reading the arguments and exiting gracefully, and changing the CTest definition to a power of 2 for NGPTOT.

Reply:
I've added a test to check whether NGPTOT and NPROMA match. I haven't added a check for the SUB_GROUP_SIZE as I expect people changing the default value to know what they are doing ... Is that ok or do you want to have another check?

Reply:
That sounds good and agreed - as opposed to NGPTOT and NPROMA, the SUB_GROUP_SIZE isn't something that's exposed as prominently to the user.